### ISSUE TITLE
[1.12.2 - 2.0.9] Client spam when connected to server

### ISSUE THREAD

**Issue by CaiganMythFang** [McJty/LostCities#137]
**Modpack** : [A Furcraftian Tail: FOGCraft Goes west](https://minecraft.curseforge.com/projects/a-furcraftian-tail-fogcraft-goes-west)
**Minecraft/Forge Version** : 1.12.2 / 2705
**Lost Cities version** : 2.0.9 (on both server and client)

**Issue** : 
When used in a server, the client log gets spammed with the following line a few hundred times a second while connected : 

> [main/INFO] [STDOUT]: [mcjty.lostcities.dimensions.world.WorldTypeTools:getProfile:74]: NO: world.provider.getDimension() = 0

The server does not have this message, nor does it happen in single player. Downgrading to Lost Cities 2.0.7 fixes this issue.

**Comment by McJty** [comment]
I'll fix this soon

**Comment by mezz** [comment]
I see this when I join a Direwolf20 server.
The spam is very heavy, I can't see other logs.

**Comment by joakime** [comment]
Can confirm, the new Direwolf20 2.0.0 update has this bug in it. (only seems to occur on client side when connected to remote server.  server does not seem to log this event)

```
[logs]$ grep "NO: world.provider.getDimension" debug.log | head -4
[10:33:13] [main/INFO] [STDOUT]: [mcjty.lostcities.dimensions.world.WorldTypeTools:getProfile:74]: NO: world.provider.getDimension() = 0
[10:33:18] [main/INFO] [STDOUT]: [mcjty.lostcities.dimensions.world.WorldTypeTools:getProfile:74]: NO: world.provider.getDimension() = 0
[10:33:18] [main/INFO] [STDOUT]: [mcjty.lostcities.dimensions.world.WorldTypeTools:getProfile:74]: NO: world.provider.getDimension() = 0
[10:33:18] [main/INFO] [STDOUT]: [mcjty.lostcities.dimensions.world.WorldTypeTools:getProfile:74]: NO: world.provider.getDimension() = 0
[logs]$ grep "NO: world.provider.getDimension" debug.log | tail -4
[10:36:33] [main/INFO] [STDOUT]: [mcjty.lostcities.dimensions.world.WorldTypeTools:getProfile:74]: NO: world.provider.getDimension() = 0
[10:36:33] [main/INFO] [STDOUT]: [mcjty.lostcities.dimensions.world.WorldTypeTools:getProfile:74]: NO: world.provider.getDimension() = 0
[10:36:33] [main/INFO] [STDOUT]: [mcjty.lostcities.dimensions.world.WorldTypeTools:getProfile:74]: NO: world.provider.getDimension() = 0
[10:36:33] [main/INFO] [STDOUT]: [mcjty.lostcities.dimensions.world.WorldTypeTools:getProfile:74]: NO: world.provider.getDimension() = 0
[logs]$ grep "NO: world.provider.getDimension" debug.log | wc -l
248445
```

So that's 248,445 entries in 3 minutes. (or about 11MB a minute)
That's some super aggressive logging, that's for sure.

**Comment by joakime** [comment]
I think this was fixed with commit 5bc3916a9993b5a12a1191a3b5fcb0ee5daabe17

**Comment by McJty** [comment]
yes, update Lost Cities
